### PR TITLE
PCHR-2112: Bug Fixes

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this ticket, bugs related to PCHR-2112 has been fixed.

## Problem 1
DOB format is still different: Android: DD/MM/YYYY, Laptop: DD-MM-YYYY

**Solution**: Changed the desktop format to match the mobile date format.

## Problem 2
The radio buttons had hardcoded width, so if the no of radio button is changed, the UI got broken.

**Solution**: Changed the radio buttons to have equal width using `display: table-cell`.
radio

## Problem 3
The radio buttons label was going outside of its boundary, incase of a long name.

**Solution**: Applied `text-overflow: ellipsis;`.
welcome_to_civihr___hr17-4

## Problem 4
The cancel button in Administer Onboarding Page was added recently and is not styled.

**Solution**: Styled the cancel button.
customize_welcome_wizard___hr17-4

---

- [x] Tests Pass

### CODE REVIEW

File: civihr_default_theme/scss/base/_variables.scss [hunk 1]
```diff
@@ -118,7 +118,7 @@ $nav-tabs-justified-active-link-border-color: $light !default;
 
 $screen-sm-max:                   767px !default;
 $crm-menu-responsive-breakpoint:  1250px;
-$crm-onboarding-wizard-menu-responsive-breakpoint:  1300px;
+$crm-onboarding-wizard-menu-responsive-breakpoint:  1170px;
```
Comment (reneolivo) [inline civihr_default_theme/scss/base/_variables.scss:121]: why was this changed to 1170px ? Why not [$screen-lg](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L314) which is 1200px?

**Review by reneolivo — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/base/_variables.scss (hunk 1 above)
Comment (deb1990) [reply civihr_default_theme/scss/base/_variables.scss:121]: The idea is not to switch to Mobile view for the Progress Bar, otherwise it is looking weird ,
welcome_to_civihr___hr17-4_

So tried to lower the breakpoint, as much as I can.

**Review by reneolivo — APPROVED** [review]
